BUILD.bazel

```diff
@@ -38,7 +29,16 @@ cc_binary(
         ":qt_qml",
         ":qt_widgets",
     ],
-    #data = [
-    #   "@qt//:platform_files",
-    #]
+    env = select({
+        "@platforms//os:linux": {
+            "QT_QPA_PLATFORM_PLUGIN_PATH": "external/qt_6.1.0_linux_desktop_gcc_64/plugins",
+        },
+        "@platforms//os:windows": {
+            # TODO
```

Comment:
Also on Windows I suspect it is better to use this approach instead of the copying you currently do, but I can't test on Windows right now.